rules/windows/privilege_escalation_port_monitor_registration.toml

```diff
@@ -0,0 +1,69 @@
+[metadata]
+creation_date = "2021/01/21"
+maturity = "production"
+updated_date = "2021/01/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+A port monitor can be configured to load a DLL at startup. An arbitrary DLL can be added to the Monitors registry path
+and executed as SYSTEM for privilege escalation or persistence, if permissions allow writing a fully-qualified pathname
+for that DLL.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Potential Port Monitor Registration Abuse"
+risk_score = 47
+rule_id = "8f3e91c7-d791-4704-80a1-42c160d7aa27"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+type = "eql"
+
+query = '''
```

Comment:
maps to the same technique, u can also add "HKLM\\SYSTEM\\*ControlSet*\\Control\\\\Print\\Environments\\Windows*\\Print Processors\\*" and registry.data.strings:"*.dll". example can be found [here](https://github.com/elastic/eql-rules/issues/1077).

Reply:
Print processors has its own separate sub technique, https://attack.mitre.org/techniques/T1547/012/. That could be split into another rule for coverage.

Reply:
both serve same goal loading an arbitrary DLL into Print Spooler Service for persistence via registry modif (it's not yet documented by MITRE as it happened last year).

Reply:
I updated the rule to also include the print processor sub technique. Its currently going to fail since we dont have T1547.012 as part of the mappings. Once we add that, it should pass.

Reply:
Spoke to @brokensound77 and were going to remove the T1547.012 sub technique for now as it would also require a Kibana change to get the Mitre info back in sync. We can add the sub technique at a later date. The rule still looks for both port monitor and print processor registry modifications.